src/github.com/getlantern/flashlight/client/client.go

```diff
@@ -215,7 +215,9 @@ func (client *Client) proxiedDialer(orig func(network, addr string) (net.Conn, e
 			log.Tracef("Rewriting %v to %v", addr, rewritten)
 			return net.Dial(network, rewritten)
 		}
+		start := time.Now()
 		conn, err := proxied(network, addr)
+		log.Debugf("Dialing proxy takes %v for %s", time.Since(start), addr)
```

Comment:
Not the initial intention for this PR, but logging dial time tells us how the client behaves from user submitted logs.